src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+  const result = [];
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
How did you know that it's 3? Don't hardcore this value. Make clear for other developers what this value is